src/vm/peimage.h

```diff
@@ -372,12 +372,32 @@ class PEImage
     DWORD m_dwMachine;
     BOOL  m_fCachedKindAndMachine;
 
-
+#ifdef FEATURE_PAL
+    static PEImageList m_preloadedAssemblies;
+#endif
 
 public:
     void CachePEKindAndMachine();
     void GetPEKindAndMachine(DWORD* pdwKind, DWORD* pdwMachine);
 
+public:
+
+#ifdef FEATURE_PAL
+    static void addNewPreloadedAssembly(LPCWSTR path, PEImageLayout *layout)
```

Comment:
Could you please follow the coding convention of the surrounding code and use pascal casing of the method names?